.github/workflows/build.yml

```diff
@@ -24,12 +24,13 @@ jobs:
       CIBW_BUILD: ${{ matrix.cibw_build }}
       CIBW_ARCHS_LINUX: ${{ matrix.arch }}
       CIBW_ARCHS_MACOS: "x86_64 arm64"
+      CIBW_FREE_THREADED_SUPPORT: true
```

Comment:
Note that this recently changed to `CIBW_ENABLE`, so if its easy to bump `cibuildwheel` to 2.22.0 then using that may be more future-proof: https://cibuildwheel.pypa.io/en/stable/changelog/#v2220